drivers/dma/dma_mcux_lpc.c

```diff
@@ -281,7 +285,7 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 	struct dma_block_config *block_config;
 	uint32_t virtual_channel;
 	uint8_t otrig_index;
-	uint8_t src_inc, dst_inc;
+	uint8_t src_inc = 0, dst_inc = 0;
```

Comment:
maybe these can start as 1. then the else blocks can be removed below

Reply:
Good idea, I will make this change.